source/production/F0.Minesweeper.Components/Logic/Cell/CellVisualisationManager.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using F0.Minesweeper.Components.Abstractions;
+using F0.Minesweeper.Components.Abstractions.Enums;
+
+namespace F0.Minesweeper.Components.Logic.Cell
+{
+	internal class CellVisualisationManager : ICellVisualisationManager
+	{
+		private static readonly Dictionary<CellStatusType, (char? content, string cssClass)> visualisationTemplates = InitializeTranslations();
+		public CellVisualisation GetVisualisation(CellStatusType cellStatus) => GetVisualisation(cellStatus, null);
+		public CellVisualisation GetVisualisation(CellStatusType cellStatus, byte? adjacentMineCount)
+		{
+			(char? content, string cssClass) template = GetVisualisationTemplateOrDefault(cellStatus);
+
+			string cssClass = string.Format(CultureInfo.InvariantCulture, template.cssClass, adjacentMineCount);
```

Comment:
For static methods, we defined in the `.editorconfig` that we prefer the Framework-Type of the Language-Keyword - so `String.Format` over `string.Format`.